maven-core/src/main/java/org/apache/maven/plugin/internal/DefaultMavenPluginManager.java

```diff
@@ -194,7 +193,17 @@ public PluginDescriptor getPluginDescriptor(
 
             PluginDescriptor descriptor = extractPluginDescriptor(pluginArtifact, plugin);
 
-            if (StringUtils.isBlank(descriptor.getRequiredMavenVersion())) {
+            boolean isBlankVersion = true;
+            if (descriptor.getRequiredMavenVersion() != null) {
+                for (int i = 0; i < descriptor.getRequiredMavenVersion().length(); i++) {
```

Comment:
just call String.trim and see if the result is empty or use SttingUtils.isBlank from Apache Commons Lang